Add a notebook to show llamaindex agent works with graphRAG and Vertex AI

wadave opened the pull request:
# Description

The sample notebook will show developers how to use:

- GraphRAG
- Llamaindex Agent
- Vertex AI

Fixes # 14772

## New Package?

Did I fill in the `tool.llamahub` section in the `pyproject.toml` and provide a detailed README.md for my new integration or package?

- [ ] Yes
- [x] No

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [x] No

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality) New Notebook
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added Google Colab support for the newly added notebooks.
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

nerdai commented on docs/docs/examples/cookbooks/agentic_graph_rag_vertex.ipynb:
```diff
@@ -0,0 +1,1130 @@
+{
```
<p>nit: <code>PropertyGraphIndex</code> (sp) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/14774/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

wadave replied:
Thanks.

nerdai reviewed (approved):
Thanks @wadave. I am wondering if maybe we should move this in `docs/examples/property_graph/`

wadave commented:
> Thanks @wadave. I am wondering if maybe we should move this in `docs/examples/property_graph/`

Sure. It makes sense.

wadave commented:
> > Thanks @wadave. I am wondering if maybe we should move this in `docs/examples/property_graph/`
> 
> Sure. It makes sense.

Done! I fixed the typo and moved to the folder

nerdai reviewed (approved):
lgtm @wadave! Thanks!
